### ISSUE TITLE
API getWanted results empty 

### ISSUE THREAD

**Issue by SenorSmartyPants** [mylar3/mylar3#1266]
**Describe the bug**
API getWanted returns empty response. Home page lists 3 issues as wanted. 

**CarePackage**
On the config page, there is a button labelled "CarePackage". Pressing this button will download a zip file containing a redacted copy of your config, your database, a summary of your operating environment, and your logs. Upload that file here.

If you're uncomfortable with pasting your CarePackage, include a DEBUG log containing the error from disk. 

Do not paste from the GUI log.
Do not just paste a traceback/error.

**Environment (please complete the following information):**
 - OS: [e.g. Windows10]
 - Branch / commit: [e.g. Dev/38aksi1]

**Additional information**
Add any other information about the problem here.

**Comment by barbequesauce** [comment]
What commit?

**Comment by SenorSmartyPants** [comment]
beb8413773b2c66e6370b676b75a33f49704f4f0

**Comment by SenorSmartyPants** [comment]
getUpcoming as well.

**Comment by barbequesauce** [comment]
Interesting - I'm on that same commit and I get respectable counts. Are yours tier 1 or 2? Wondering what else could be the differentiator:

``` 
➜  mylar3 git:(python3-dev) curl http://pkf:9095/mylar/api\?apikey\=1baddad\&cmd\=getWanted | grep -o ":" | wc -l
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  5840  100  5840    0     0  45824      0 --:--:-- --:--:-- --:--:-- 49914
     207
➜  mylar3 git:(python3-dev) curl http://pkf:9095/mylar/api\?apikey\=1baddad\&cmd\=getUpcoming | grep -o ":" | wc -l
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   712  100   712    0     0   6069      0 --:--:-- --:--:-- --:--:--  6846
      31
➜  mylar3 git:(python3-dev) 
```

**Comment by evilhero** [comment]
When you say home page, do you mean the Wanted page? And if it's that page zis it in the upper table or the lower table, as the lower table are issues that don't have any CV info yet but they're marked as Wanted (so there would be no response in the api call as there is no info to attach to it)

**Comment by SenorSmartyPants** [comment]
Yes. The wanted page.
![image](https://user-images.githubusercontent.com/991618/182487339-8a8b0471-72e6-4379-881b-e649e8dc1c35.png)

**Comment by SenorSmartyPants** [comment]
curl "http://rockpi:8090/mylar/api?apikey=apikey&cmd=getWanted"  | grep -o ":" | wc -l
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     2  100     2    0     0     76      0 --:--:-- --:--:-- --:--:--    80
0

**Comment by evilhero** [comment]
Those release dates don't exist.. Issue q of action comics is from a few months ago, as well as Batman? Sounds like you got some issues that got disjointed from whatever series they were trying to be attached to

**Comment by SenorSmartyPants** [comment]
https://comicvine.gamespot.com/new-comics/?company=10&startWeek=06%2F01%2F2022

Those are the release dates of Batman and AC annuals, they release on Tuesdays now.

**Comment by evilhero** [comment]
Have you tried refreshing those series? 

They should have the correct naming of the annuals, not just the series it's attached to.

It really sounds like those issues got pulled down from CV when they didn't have any info and then it couldn't update the series for whatever reason (CV being down for a whole day almost would cause it to miss updating that series if it got updated during that outage)

**Comment by barbequesauce** [comment]
OK, I can repro it - seems to be specific to issue #1. More to come

**Comment by SenorSmartyPants** [comment]
I refreshed the series just now. Name on Wanted page is still just the series name (not the annual), however on the series page the annuals have the correct names where they are listed on the bottom of the page.

**Comment by SenorSmartyPants** [comment]
All 3 of those wanted issues are annuals.

**Comment by evilhero** [comment]
Both problems should be now fixed in #1271.

Note that the JSON response for this endpoint has changed in order to accommodate the different Wanted types (issues, annuals, storyarcs). 

See [here](https://github.com/mylar3/mylar3/wiki/getWanted) for additional information
